docs/guides/platform/manager/dns-manager-transfer-domain-zone-shortguide/index.md

```diff
@@ -36,3 +36,9 @@ Granting another server access to zone information is potentially dangerous. Do
     {{< note >}}
 When the DNS servers no longer need access to your Domain's zone file, remove the IP address from the **Domain Transfers** field.
     {{</ note >}}
+
+When performing the AXFR DNS query, point your secondary name server to axfr1.linode.com (or up to axfr5.linode.com) instead of ns1.linode.com. To test the AXFR query locally, follow the above instructions to allow your computer's IP address as one of the ****Domain Transfer IPs**** in the SOA Record for your domain. This may take a few minutes before going into effect. Then run the following `dig` command, replacing **example.com** with your domain:
+
+dig axfr example.com @axfr1.linode.com
```

Comment:
Looks like this  needs to be indented four  spaces to make it standout as a command. This  might  also   clear up the Vale  error on the PR.

Reply:
Just made that revision in the most recent commit